package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@elastic.io/component-commons-library",
-  "version": "1.1.6",
+  "version": "1.1.7",
```

Comment:
```suggestion
  "version": "1.1.7.dev-1",
```

Reply:
1.1.7-dev.3
There are 2 dev-versions on NPM already

Reply:
And run `npm i` to update version in `package-lock.json`

Reply:
done